src/find/azext_find/custom.py

```diff
@@ -46,8 +48,27 @@ def process_query(question):
                 print('\033[1m' + current_title + '\033[0m')
                 print(current_snippet)
 
-                if i + 1 < num_results_to_show:
-                    print("\n")
+                print("")
+            feedback = prompt("[\033[1mEnter to close.\033[0m Press + or - to give feedback]:")
+            if feedback in ['+', '-']:
+                print('Wow, you are a true hero!')
+                print("""\
+        O_
+       \033[1m<T>\033[0m`-.
+        \033[1m|\033[0m`-‘
+        \033[1mI\033[0m
+                        """)
+                print('My human overloads review each of these reports; I\'m told these reports makes me smarter.')
+                print('Send us more feedback by email: aladdindoc@microsoft.com')
+            properties = {}
+            set_custom_properties(properties, 'Feedback', feedback)
+            telemetry_core.add_extension_event(EXTENSION_NAME, properties)
+
+
+def set_custom_properties(prop, name, value):
+    if name and value is not None:
+        # 10 characters limit for strings
+        prop['{}{}'.format(ALIAS_EXTENSION_PREFIX, name)] = value[:10] if isinstance(value, str) else value
```

Comment:
Why is there a 10 char limit for feedback? Seems a little short.

Reply:
In practice we want them to only type + or -. It's not open ended feedback. Please see the screenshot above.